config/initializers/assets.rb

```diff
@@ -5,35 +5,3 @@
 
 # Add Yarn node_modules folder to the asset load path.
 Rails.application.config.assets.paths << Rails.root.join("node_modules")
-
-# Precompile additional assets.
-# application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
-# Rails.application.config.assets.precompile += %w( search.js )
-Rails.application.config.assets.precompile += %w[
-  admin.css
-  admin-ie8.css
-  admin-ie7.css
```

Comment:
admin-ie8 and admin-ie7 still exist - do they need to be deleted? (I'm assuming they're unused).

Reply:
Believe it or not, they're actually still there in the HTML:

```
<!--[if gt IE 8]><!--><link rel="stylesheet" media="screen" href="/assets/whitehall/admin-06bf101f04b43ec37f6bdee1a0b6ac5fbea5a306a6fedce5957a61ce1d8e6eba.css" /><!--<![endif]-->
--
  | <!--[if IE 7]><link rel="stylesheet" media="screen" href="/assets/whitehall/admin-ie7-b226854368821de0821c264f4a6ec977dfea79121c19836d9ccb1c424b64110a.css" /><script>var ieVersion = 7;</script><![endif]-->
  | <!--[if IE 8]><link rel="stylesheet" media="screen" href="/assets/whitehall/admin-ie8-99a031ca70cfa19de4e82ade599c5770eedd4c213deb9fd58fe4e5c117662ca0.css" /><script>var ieVersion = 8;</script><![endif]-->
```

Reply:
I feel like we should be able to remove them fully, however the `admin.scss` is riddled with references that generate these files (of the form `@include ie-lte(8) { }`), so maybe leave this as is for now and we can take another look at removing them later.